### DISCUSSION TITLE
Content is painted over another window

### DISCUSSION THREAD

**Discussion by RobbeCl** [i3/i3#5690] [Q&A]
I have no idea if this is due to i3wm or my graphic card. I don't know how to debug this one. 
Does anyone have an idea what this could be?

I typed 2 commands (2 times "man mn") in a row and the content of the second run seems to be painted above the other one)

The content is not only painted over in terminals, it is everywhere and also rebooting does not solve it.

![IMG_20230925_082229](https://github.com/i3/i3/assets/25816740/434087f5-5ae2-4500-b494-6b80719ac588)

Thanks!

**Comment by orestisfl** [comment]
> The content is not only painted over in terminals, it is everywhere and also rebooting does not solve it.

You mean the text remains even after reboots and is shown over other applications? This is a serious indication that something is wrong with your graphics card.

I would, just in case, stop any compositor from running, uninstall any graphics driver and check if the problem persists with a different window manager.
